packages/experimental/src/smart-session/utils/CosignerService.ts

```diff
@@ -128,11 +128,11 @@ export class CosignerService {
     const response = await sendCoSignerRequest<
       AddPermissionRequest,
       AddPermissionResponse,
-      { projectId: string }
+      { projectId: string; v: string }
     >({
       url,
       request: data,
-      queryParams: { projectId: this.projectId },
+      queryParams: { projectId: this.projectId, v: '2' },
```

Comment:
In the `grantPermissions` file we do use `'2.0'`, and here it's `'2'` - is it okay?